### ISSUE TITLE
Cmd unable to read private key via ssl command

### ISSUE THREAD

**Issue by ravifulani** [openssl/openssl#18457]
I am using below command, but i get error unable to read private key.

    ```
    cd "C:\Program Files\OpenSSL-Win64\bin"
openssl.exe req -new -x509 -key C:\Users\Ashley\Desktop\privkey.pem -out C:\Users\Ashley\Desktop\cacert.pem -days 1095
    ```

-->

**Comment by t8m** [comment]
Do you have a private key in C:\Users\Ashley\Desktop\privkey.pem ?

**Comment by nhorman** [comment]
marking as inactive, to be closed at the end of 3.4 dev barring further input
